documentation/book/assembly-zookeeper-replicas.adoc

```diff
@@ -0,0 +1,40 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-zookeeper-replicas-{context}']
+
+= Replicas
+
+Zookeeper clusters / ensembles usually run with odd number of nodes.
```

Comment:
Zookeeper clusters / ensembles usually run with odd number of nodes.
It always requires the majority of the nodes to be available in order to maintain quorum.

TO

Zookeeper clusters or ensembles usually run with an odd number of nodes and always requires the majority of the nodes to be available in order to maintain a quorum.